test.js

```diff
@@ -143,6 +143,32 @@ test("RPC and message passing", function(t) {
   });
 });
 
+test("RPC Call without passing any message/data", function(t) {
+  t.plan(5);
+
+  const bs = new Bugout({dht: false, tracker: false});
+  const bc = new Bugout(bs.address(), {dht: false, tracker: false});
+
+  bs.register("call_without_args", function(pk,args){
+    console.dir(args)
+    t.assert(args === null , 'args should be null , when called without args from another peer')
```

Comment:
Small typo: there shouldn't be a space after the word "null".

Reply:
correct :)